### PR TITLE
Updating README to match new folder structure

### PR DESCRIPTION
**Pull request by NeerajW**
1. Added a table of contents.
2. Reorganized the readme so its easier to navigate.
3. Update the readme with appropriate links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,56 @@
...
-
-## Plugins
-Folder containing the native and managed binaries being compiled from the HoloToolkit repository.
-
----
+HoloToolkit contains the following feature areas:
 
-## Utilities
-Useful common concepts that you can leverage in your apps.
+1. [Input](https://github.com/NeerajW/HoloToolkit-Unity#input)
```
Comment (keveleigh) [inline README.md]: These links go to your forked repo (https://github.com/NeerajW/HoloToolkit-Unity), but should link to the main Microsoft repository (https://github.com/Microsoft/HoloToolkit-Unity).

File: README.md [hunk 2]
```diff
@@ -104,37 +114,116 @@ or in your Visual Studio Package.appxmanifest capabilities.
...
 **RecognizerStart** Set this to determine whether the keyword recognizer will start immediately or if it should wait for your code to tell it to start.
 
----
+### [Tests](https://github.com/Microsoft/HoloToolkit-Unity/tree/master/Assets/HoloToolkit/Input/Tests)
+Tests related to the input features. To use the scene:
+1. Navigate to the Tests folder.
+2. Double click on the test scene you wish to explore.
+3. Either click "Play" in the unity editor or File -> Build Settings.
+4. Add Open Scenes, Platform -> Windows Store, SDK -> Universal 10, Build Type -> D3D, Check 'Unity C# Projects'.
+5. CLick 'Build' and create an App folder. When compile is done, open the solution and deploy to device.
```
Comment (keveleigh) [inline README.md]: Typo: CLick

File: README.md [hunk 3]
```diff
@@ -104,37 +114,116 @@ or in your Visual Studio Package.appxmanifest capabilities.
...
+Utility scripts for the Sharing.prefab.
+Also scripts for logging, launching processes, math utilities.
+
+### [Tests](https://github.com/Microsoft/HoloToolkit-Unity/tree/master/Assets/HoloToolkit/Sharing/Tests)
+Tests related to the sharing features. To use the scene:
+1. Navigate to the Tests folder.
+2. Double click on the test scene you wish to explore.
+3. Either click "Play" in the unity editor or File -> Build Settings.
+4. Add Open Scenes, Platform -> Windows Store, SDK -> Universal 10, Build Type -> D3D, Check 'Unity C# Projects'.
+5. CLick 'Build' and create an App folder. When compile is done, open the solution and deploy to device.
```
Comment (keveleigh) [inline README.md]: Typo: CLick

File: README.md [hunk 4]
```diff
@@ -144,16 +233,18 @@ This component is for doing physics against the Spatial Mapping mesh.
 
 **Radius**: (Default: **10**). The radius of the bounding sphere in meters. Only applicable with Sphere set for Bounding Volume.
 
-**Level of Detail**: (Default: **Low**). The amount of detail in the resulting meshes. Possible values are Low, Medum, and High. Higher levels of detail will result in lower performance.
+**Level of Detail**: (Default: **Medium**). The amount of detail in the resulting meshes. Possible values are Low, Medum, and High. Higher levels of detail will result in lower performance.
```
Comment (keveleigh) [inline README.md]: Typo: Medum

File: README.md (hunk 1 above)
Comment (NeerajW) [inline README.md]: Fixed. Thanks for catching this.

File: README.md [hunk 5]
```diff
@@ -271,75 +363,120 @@ Generates planes and classifies them by type (wall, ceiling, floor, table, unkno
...
+#### VertexLitConfigurable.cginc
+Code shared between VertexLitConfigurable.shader and VertexLitConfigurableTransparent.shader.
+
+#### VertexLitConfigurable.shader
+Feature configurable vertex lit shader.  Use when a higher performance but lower precision lighting trade-off is acceptable.
+
+#### VertexLitConfigurableTransparent.shader
+Feature configurable vertex lit transparent shader.  Use when a higher performance but lower precision lighting trade-off is acceptable, and transparency is needed.
+
+#### Occlusion.shader
```
Comment (keveleigh) [inline README.md]: The Occlusion and Wireframe shaders have moved under SpatialMapping

File: README.md (hunk 4 above)
Comment (NeerajW) [inline README.md]: Fixed.

**Pull request merged**